src/components/Messages/MessageStyles.js

```diff
@@ -5,7 +5,7 @@ export const StyledButton = styled('button')({
   height: '60px',
   backgroundColor: '#017969',
   borderColor: 'black',
-  borderRadius: '5px',
+  borderRadius: '25px',
```

Comment:
Why are we not using our MUI button that we are adding these default styles to?

https://github.com/codeforpdx/PASS/blob/8dd714b95c3748bc50545438e9347e0a363e38a5/src/theme.js#L58-L64

Reply:
Is this styled button just a relic from another time? Just wondering if we can fold it all together so we don't have multiple same-but-different buttons

Reply:
If not in this PR, maybe a separate issue to keep things clean

Reply:
> Is this styled button just a relic from another time? Just wondering if we can fold it all together so we don't have multiple same-but-different buttons

Have just checked, outside of the `Pagination Container`, the rest of the styles are relics from an earlier styling scheme.

Think we could remove them here, but we could also open up a fresh PR just to have them removed

Reply:
cleared code that is no longer used and renamed file to `PaginationStyles.js`